qa/automation/ScoringPortal.Tests.Scenarios/3.Sign_in_with_GoogleAcc/1. sign_in.feature

```diff
@@ -0,0 +1,36 @@
+Feature: 3.1 Sign in with Google Account
+
+Background:
+	Given user turn on VPN with necessary config
+	And go to https://qa.ddso-spot.quantori.com/
+
+Scenario: 3.1_01 User has Google account
+	Given User has a saved Google account
+	When User press the button "Sign in with Google"
+	And User select necessary Google account 
+	Then the page with Quantori Scoring is opened
+
+Scenario: 3.1_02 User doesn't have a saved Google account
+	Given User has a google account but it isn't saved in the browser
+	When User press the button "Sign in with Google"
+	And User see the "Sign in to your Google account" field
+	And User fills in all necessary information
+	Then the page with Quantori Scoring is opened
+
+Scenario: 3.1_03 User doesn't have any Google account
+	Given User doesn't have any Google account
+	When User press the button "Sign in with Google"
+	And User see the "Sign in to your Google account" field
+	And fills in the e-mail which is not correspond to any Google account
+	Then User see the red "Could not find google account" message
+	
+	# There is no mention about possibility to change Google accounts, but I decide it should be possible and test it. 
```

Comment:
It is possible on Google homepage, I made couple of test Google accounts for this task and logged in and out of them during testing. Try it too. Hint: Incognito mode helps a lot.

Reply:
Also try to put some testing information you use in Examples: in scenario.

Reply:
I didn't test google autorization system, only implementation with . When I use incognito page in  google chrome I can log in, then log out and after that, on the same incognito page, I can only log in with the first google account

Reply:
And what about Examples?